.doc_gen/metadata/cognito-identity-provide_metadata.yaml

```diff
@@ -0,0 +1,17 @@
+# zexi 0.4.0
+cognito-identity-provider_ListUserPools:
+  title: List the &COGIDP; associated with an &AWS; account using an &AWS; SDK
```

Comment:
This one expands weird...I think this might be better:
`List &COGUP; using an &AWS; SDK`

Reply:
Done